input_topic: "/laser_point_cloud"
verbose: true
debug: false
directory: "/home/wu/Documents/biorobotics/recording/pcds/"

cropFilterOn: true
XYZpassMin:
 - -0.036992848 
 - -0.473493063
 - 0.404776374
XYZpassMax:
 - 0.263007152
 - -0.204493063
 - 0.704776374

SORon: false
meanK: 50
stdThresh: 1

voxelFilterOn: true
leafSize: 0.0005


